### PR TITLE
Follow up to earlier PR

### PR DESCRIPTION
**Pull request by shayne-fletcher**
Follow up on  https://github.com/digital-asset/daml/pull/5171.

### CODE REVIEW

File: language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs [hunk 1]
```diff
@@ -174,57 +191,43 @@ tests damlTypes yarn damlc daml2ts davl = testGroup "daml2ts tests"
...
-        assertFileExists (daml2tsDir </> "davl-0.0.4" </> "lib" </> "DAVL.js")
-        assertFileExists (daml2tsDir </> "davl-0.0.4" </> "lib" </> "DAVL.d.ts")
-        assertFileExists (daml2tsDir </> "davl-0.0.5" </> "src" </> "DAVL.ts")
-        assertFileExists (daml2tsDir </> "davl-0.0.5" </> "lib" </> "DAVL.js")
-        assertFileExists (daml2tsDir </> "davl-0.0.5" </> "lib" </> "DAVL.d.ts")
-        assertFileExists (daml2tsDir </> "davl-upgrade-v4-v5-0.0.5" </> "src" </> "Upgrade.ts")
-        assertFileExists (daml2tsDir </> "davl-upgrade-v4-v5-0.0.5" </> "lib" </> "Upgrade.js")
-        assertFileExists (daml2tsDir </> "davl-upgrade-v4-v5-0.0.5" </> "lib" </> "Upgrade.d.ts")
+        mapM_ (assertTsFileExists (daml2tsDir </> "davl-0.0.4")) [ "index", "DAVL" ]
+        mapM_ (assertTsFileExists (daml2tsDir </> "davl-0.0.5")) [ "index", "DAVL" ]
```
Comment (hurryabit) [inline language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs]: I'm surprised this test actually passes. There is no `DAVL.ts` in v5, there's only `DAVL/V5.ts`. Can you please investigate?

File: language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs [hunk 2]
```diff
@@ -262,5 +265,11 @@ tests damlTypes yarn damlc daml2ts davl = testGroup "daml2ts tests"
           , "resolutions" .= HMS.fromList ([("@daml/types", "file:daml-types")] :: [(T.Text, T.Text)])
           ]
 
-    assertFileExists :: FilePath -> IO ()
-    assertFileExists file = doesFileExist file >>= assertBool (file ++ " was not created")
+    assertTsFileExists :: FilePath -> String -> IO ()
+    assertTsFileExists proj file = do
+      assertFileExists (proj </> "src" </> file <.> "ts")
+      assertFileExists (proj </> "lib" </> file <.> "js")
+      assertFileExists (proj </> "lib" </> file <.> ".d.ts")
```
Comment (hurryabit) [inline language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs]:
```suggestion
      assertFileExists (proj </> "lib" </> file <.> "d.ts")
```

File: language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs (hunk 1 above)
Comment (shayne-fletcher) [reply language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs]: Indeed, it's there : https://raw.githubusercontent.com/digital-asset/davl/51d3977be2ab22f7f4434fd4692ca2e17a7cce23/v5/daml/DAVL.daml. I can only suppose that v5 changed in more recent commits?

Comment (hurryabit) [reply language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs]: Oh, that explains it. Thanks. Let's move to a newer version of DAVL in another PR.

Comment (shayne-fletcher) [reply language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs]: You got it. I shall take care of it.

**Pull request merged**